core/primitives/src/views.rs

```diff
@@ -230,24 +230,27 @@ pub struct GasKeyView {
     pub num_nonces: NonceIndex,
     pub balance: Balance,
     pub permission: AccessKeyPermissionView,
+    /// If requested, the nonces are included.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub nonces: Option<Vec<Nonce>>,
```

Comment:
Would it be cleaner to either always include nonces or have a separate request for nonces only? I don't fully understand why we want to have it optional and having it optional seems inconvenient - yet another option to keep in mind.